contrib/sort.py

```diff
@@ -95,6 +96,8 @@ def fix_profile(filename):
                 fixed_line = line
             if fixed_line != line:
                 was_fixed = True
+                if getenv("CI"):
+                    print(f"{filename}:{lineno + 1}:{fixed_line}")
```

Comment:
Why print only on CI?

I would like to see what it does/does not when running locally.  Also, I like
the output of this print call as it is similar to what is done by both the
linter and the other script that I mentioned [here][1] (except that sort.py
prints the fixed version rather than the original one).

For example, I have been working on a PR that (among other things) adds a
makefile target for running sort.py (to then add the aforementioned linter on
it), so that it's easier to run locally and one of my gripes with it is the
lack of feedback when there are no errors.  For example, when messing with the
makefile and running the target, I wasn't sure if sort.py was even being
executed and if it was, if it did actually check the profiles that I intended
to check or if I passed an empty/wrong var.

It would be nice to see something like this at the beginning:

```
sort.py: checking 1001 profiles...
```

Example in shell (mentally replace it with the python equivalent):

```sh
$ printf '%s: checking %s profiles...\n' "$(basename "$0")" $#
```

Full output example:

```
sort.py: checking 1001 profiles...
etc/profile-a-l/ark.profile:39:private-bin 7z,aaa,ark,bash,lrzip,lsar,lz4,lzop,p7zip,rar,sh,tclsh,unar,unrar,unzip,zip,zipinfo
[ Fixed ] etc/profile-a-l/ark.profile
etc/profile-m-z/mpv.profile:75:private-bin aaa,env,mpv,python*,waf,youtube-dl
[ Fixed ] etc/profile-m-z/mpv.profile
```

Though this other print could be left for another PR of course.

Other than that, LGTM.  The commits are looking nice too.

[1]: https://github.com/netblue30/firejail/issues/3979#issuecomment-778491722